_tools/types/main.go

```diff
@@ -0,0 +1,341 @@
+// Generates a report about every driver wich UASTv2 types does it use,
+// both in actual test fixtures and though the source code.
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/bblfsh/sdk/driver/manifest/discovery"
+	"github.com/bblfsh/sdk/uast"
+
+	_ "net/http/pprof"
+)
+
+const (
+	fixtureDir = "fixtures"
+	fixtureExt = ".sem.uast"
+	goDocURL   = "https://godoc.org/github.com/bblfsh/sdk/uast#"
+)
+
+var (
+	reposRootPath = path.Join(".", "drivers")
+	uastNodeRe    = regexp.MustCompile("uast:[^\"]*")
+
+	pprof      = flag.Bool("pprof", false, "start pprof profiler http endpoing")
```

Comment:
```suggestion
	pprof      = flag.Bool("pprof", false, "start pprof profiler http endpoint")
```

Consider allowing the user to set the address for this, or at least document where it will run so they can tell from the `-help` text:

```go
   pprofPort = flag.Int("pprof", 0, "If non-zero start a pprof profiler HTTP endpoint at this port")

if *pprofPort > 0 {
   pprofAddr := fmt.Sprintf("localhost:%d", *pprofPort)
   …
}
```

or words to that effect?
```

Reply:
We usually have a separate flag for overriding the port, but as it's a small script I figured we can get by with the default port and a simpler usage, as soon as port is explicitly printed for convenience in terminal.

As it is meant to be run by `go run ..` - I would argue that it's alright to expect anyone interested in changing the port to just edit the default in line 44.

But please, let me know if you think we should add a add port overriding option though.

Reply:
What you say makes sense. I agree we probably don't need to change the flag. But could you please include the port in the help text, so someone doesn't have to run the tool "for real" just to find out what the port is? E.g., `"Start a pprof profiler HTTP service at port 6060"` or words to that effect?

Reply:
Added in 67e7dd961bccbd47efe815503e9b3882cc114c7c